spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/reader/KeyValueRowReader.scala

```diff
@@ -1,24 +1,18 @@
 package com.datastax.spark.connector.rdd.reader
 
 import com.datastax.driver.core.{ProtocolVersion, Row}
-import com.datastax.spark.connector.ColumnRef
+import com.datastax.spark.connector.{ColumnSelector, ColumnRef}
 import com.datastax.spark.connector.cql.TableDef
-import com.datastax.spark.connector.mapper.TupleColumnMapper
 
-class KeyValueRowReaderFactory[K, V](keyRRF: RowReaderFactory[K], valueRRF: RowReaderFactory[V])
+class KeyValueRowReaderFactory[K, V](
```

Comment:
I wonder about things like there, where it is not a private[connector] and thus a public API and we modify the API over time. I like the mods but wonder if it needs to be public?